app/code/Magento/Contact/Test/Unit/Model/ConfigTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Contact\Test\Unit\Model;
+
+use Magento\Contact\Model\Config;
+use PHPUnit\Framework\TestCase;
+use PHPUnit\Framework\MockObject\MockObject;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+
+/**
+ * Unit test for Magento\Contact\Model\Config
+ */
+class ConfigTest extends TestCase
+{
+    /**
+     * @var Config
+     */
+    private $model;
+
+    /**
+     * @var ScopeConfigInterface|MockObject
+     */
+    private $scopeConfigMock;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp(): void
+    {
+        $this->scopeConfigMock = $this->getMockBuilder(ScopeConfigInterface::class)
+            ->setMethods(['getValue', 'isSetFlag'])
+            ->disableOriginalConstructor()
+            ->getMockForAbstractClass();
+
+        $objectManager = new ObjectManagerHelper($this);
+        $this->model = $objectManager->getObject(
+            Config::class,
+            [
+                'scopeConfig' => $this->scopeConfigMock
+            ]
+        );
+    }
+
+    /**
+     * Test isEnabled()
+     *
+     * @return void
+     */
+    public function testIsEnabled(): void
+    {
+        $this->scopeConfigMock->expects($this->once())
```

Comment:
No need to duplicate the logic for `testIsEnabled` and `testIsNotEnabled`. Please use a data provider instead.

Reply:
@dmytro-ch Thanks for the feedback, I have removed `testIsNotEnabled` and added data provider.